### PR TITLE
DOC-5711 - current docs cleanup

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-5711

Changes:

- move import processing paragraph to `import_docs` prop (from `enable_shared_bucket_access`)
- break first paragraph in `import_docs` intro
- review paragraphs under `import_docs` (Tombstones and Implementation notes for XATTRs)
- fix up external links in the `enable_shared_bucket_access` description

### CODE REVIEW

**Comment by jamesnocentini** [comment]
@rajagp For your information, this PR is a small clean up of current docs. Please let me know if you have any feedback on this. Many thanks.

File: modules/ROOT/assets/attachments/sg.yaml [hunk 1]
```diff
@@ -142,9 +142,33 @@ properties:
...
             type: boolean
             description: |
-              This property works in conjunction with the [enable_shared_bucket_access](#databases-foo_db-enable_shared_bucket_access) property. When **shared bucket access** is enabled, setting this property to true (`"import_docs": true`) indicates that this Sync Gateway node should perform import processing.
+              Introduced in Sync Gateway 1.5, this property specifies whether this Sync Gateway node should perform import processing.
+
+              This property works in conjunction with the [enable_shared_bucket_access](#databases-foo_db-enable_shared_bucket_access) property.
 
               `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
+
+              Starting Sync Gateway with `import_docs` enabled will generate the mobile-specific metadata for all the pre-existing documents in the Couchbase Server bucket. From then on, documents can be inserted on the Server directly (with N1QL or SDKs) or through the Sync Gateway REST API.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]:
```suggestion
              Starting Sync Gateway with `import_docs` enabled will generate the mobile-specific metadata for all the existing documents in the Couchbase Server bucket. With shared_bucked_access enabled, documents can be mutated on the Server directly (with N1QL or SDKs) or through the Sync Gateway REST API and these changes will be seamlessly synchronized over to the connected Couchbase Lite enabled clients.
```

File: modules/ROOT/assets/attachments/sg.yaml [hunk 2]
```diff
@@ -142,9 +142,33 @@ properties:
...
+              The document is first run through the Sync Function to compute read security and routing, with the following differences:
+
+              - The import is processed with an admin user context in the Sync Function, similar to writes made through the Sync Gateway Admin API. This means that `requireAccess`, `requireUser` and `requireRole` calls in the Sync Function are treated as no-ops.
+              - During import, `oldDoc` is `nil` when the Sync Function is executed.
+
+              Only one node in a Sync Gateway cluster should be configured for feed-based import - it processes the full DCP stream.
+
+              The [Import+](#1.5/log) log key can be used to troubleshoot the import process further in the logs.
+
+              A filter function can be specified using the [databases.foo_db.import_filter](#1.5/databases-foo_db-import_filter) property to only import specific documents.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]: The entire "Import Process" section should be moved to https://docs.couchbase.com/sync-gateway/2.6/shared-bucket-access.html. Its a concept that we don't want buried in config specs. Also, please rename "Mobile-Web Data Sync" navigation menu to "Mobile-Server Data Sync"

File: modules/ROOT/assets/attachments/sg.yaml [hunk 3]
```diff
@@ -142,9 +142,33 @@ properties:
...
+
+              ##### Import and Sync Gateway Accelerator
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]: Remove this section. SG Accelerator is REMOVED and not a product offering. Please remove all references in docs

File: modules/ROOT/assets/attachments/sg.yaml [hunk 4]
```diff
@@ -142,9 +142,33 @@ properties:
...
+
+              In a deployment with Sync Gateway Accelerator, feed-based import processing is performed by the Accelerator nodes. Import work is sharded across the Accelerator cluster, in the same way that indexing work is sharded today.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]: Remove all references to Accelerator. Its not a product offering

File: modules/ROOT/assets/attachments/sg.yaml [hunk 5]
```diff
@@ -304,43 +328,26 @@ properties:
...
-              `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
-
-              On start-up, Sync Gateway will generate the mobile-specific metadata for all the pre-existing documents in the Couchbase Server bucket. From then on, documents can be inserted on the Server directly (with N1QL or SDKs) or through the Sync Gateway REST API.
-
-              #### Import process
-
-              Import is the name for the process by which non-Sync Gateway writes obtain mobile metadata. Any non-Sync Gateway write is eligible for import. A filter function can be specified using the [databases.foo_db.import_filter](#1.5/databases-foo_db-import_filter) property to only import specific documents.
-
-              The document is first run through the Sync Function to compute read security and routing, with the following specifics/differences:
+              Introduced in Sync Gateway 1.5, this property specifies whether to enable Mobile-Web Data Sync.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]:
```suggestion
              Introduced in Sync Gateway 1.5, this property specifies whether to enable Mobile-Server Data Sync a.k.a "mobile convergence". You can learn more about this functionality [here](https://docs.couchbase.com/sync-gateway/current/shared-bucket-access.html)
```

File: modules/ROOT/assets/attachments/sg.yaml [hunk 6]
```diff
@@ -304,43 +328,26 @@ properties:
...
 
-              In a deployment with Sync Gateway Accelerator, feed-based import processing is performed by the Accelerator nodes. Import work is sharded across the Accelerator cluster, in the same way that indexing work is sharded today.
+              `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
 
               #### Tombstones
 
-              When convergence is enabled, mobile tombstones are now also server tombstones. The document body is deleted, and the information required to replicate the tombstone is retained in the mobile extended attribute.
-
-              ##### Metadata Purge Interval
+              When `enable_shared_bucket_access` is enabled, mobile tombstones are now also server tombstones. The document body is deleted, and the information required to replicate the tombstone is retained in the mobile extended attribute.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]:
```suggestion
              When `enable_shared_bucket_access` is enabled, mobile tombstones are now also server tombstones. The document body is deleted, and only the mobile sync metadata required to replicate the tombstone is retained in the mobile extended attribute.
```

Comment (rajagp) [reply modules/ROOT/assets/attachments/sg.yaml]: Move the "tombstone" section over to https://docs.couchbase.com/sync-gateway/2.6/shared-bucket-access.html. Its a core concept

File: modules/ROOT/assets/attachments/sg.yaml [hunk 7]
```diff
@@ -304,43 +328,26 @@ properties:
...
 
               The server's metadata purge interval becomes an important consideration for mobile deployments under convergence. When the server purges a tombstone (based on the metadata purge interval), that tombstone will no longer be replicated to mobile clients.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml:314]: Move the Metadata Purge Interval Section to https://docs.couchbase.com/sync-gateway/2.6/shared-bucket-access.html

File: modules/ROOT/assets/attachments/sg.yaml [hunk 8]
```diff
@@ -304,43 +328,26 @@ properties:
...
 
-              For most mobile deployments, customers should set the server's metadata purge interval based on their expected client replication frequency, to ensure that clients are notified of the tombstone prior to that tombstone being purged.
+              For most mobile deployments, users should set the server's metadata purge interval based on their expected client replication frequency, to ensure that clients are notified of the tombstone prior to that tombstone being purged.
```
Comment (rajagp) [inline modules/ROOT/assets/attachments/sg.yaml]:
```suggestion
              Users should set the server's metadata purge interval based on their expected client partition window, to ensure that clients are notified of the tombstone prior to that tombstone being purged on the server.
```

**Review by rajagp — DISMISSED** [review]
@jamiltz  Please note that some of the content needs to be moved out to a different page. This is important as these sections are impacted by the "write scale enhancements" that are in place for 2.7

File: modules/ROOT/assets/attachments/sg.yaml (hunk 2 above)
Comment (ibsoln) [reply modules/ROOT/assets/attachments/sg.yaml]: Done in PR#140 https://github.com/couchbase/docs-sync-gateway/pull/140/files/0feceacb42434bab2415d837210010ca0f4f8fb4

**Review by ibsoln — APPROVED** [review]

**Review by ibsoln — APPROVED** [review]
Changed as requested

**Review by ibsoln — APPROVED** [review]

**Pull request merged**